app/gateway/2.7.x/configure/auth/oidc-okta.md

```diff
@@ -9,84 +9,209 @@ For information about configuring OIDC using Okta as an Identity provider
+  --data name="openid-connect"                                                                             \
   --data config.issuer="https://YOUR_OKTA_DOMAIN/oauth2/YOUR_AUTH_SERVER/.well-known/openid-configuration" \
-  --data config.client_id="YOUR_CLIENT_ID" \
-  --data config.client_secret="YOUR_CLIENT_SECRET" \
-  --data config.redirect_uri="https://kong.com/api" \
-  --data config.scopes="openid" \
-  --data config.scopes="email" \
+  --data config.client_id="YOUR_CLIENT_ID"                                                                 \
+  --data config.client_secret="YOUR_CLIENT_SECRET"                                                         \
+  --data config.redirect_uri="https://kong.com/api"                                                        \
+  --data config.scopes="openid"                                                                            \
+  --data config.scopes="email"                                                                             \
   --data config.scopes="profile"
 ```
 
-Some of the configurations above must use values specific to your environment:
+For a list of all available configuration parameters and what they do, see the
+[OIDC plugin reference](/hub/kong-inc/openid-connect).
 
-* The `issuer` URL can be found from your Authorization Server settings.
-* The `redirect_uri` should be the URI you specified earlier when configuring your app.
-You can view and edit this from the **General** page for your application.
-* For `client_id` and `client_secret`, replace `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET`
-with the client ID and secret shown in your Okta application's **General** page.
+{% endnavtab %}
+{% endnavtabs %}
 
 Visiting a URL matched by that route in a browser will now redirect to Okta's authentication
 site and return you to the redirect URI after authenticating.
 
-Additional plugin parameter to consider:
+### Test Your Configuration
+
+Test the following conditions to ensure a successful integration of the OIDC plugin with Okta. Depending on the route you are using, the actual commands might vary. The following is an example of testing your configuration.
 
-* The `auth_methods` parameter defines a lists of all the authentication methods
-that you want the plugin to accept. By default, its value is a list of all the supported methods.
-It is advisable to define this parameter with only the methods you want to allow.
+* Unauthorized access to a route is blocked
+* Authorized access is allowed after login/providing first set of credentials
+    * Ensure the identity is registered with the IdP
+    * Steps for debugging
+* Authorized access is allowed on immediate subsequent attempts
+    * Perhaps highlight where the cookie is stored
+* Previously authorized access is no longer allowed after cookie terminates
+    * Set a very short TTL on the session to ensure the tester doesn’t need to wait long to get locked out
 
 ### Access Restrictions
 
```

Comment:
Putting this here bc GH won't let me tag unchanged lines. Recommend changing "the configuration above" on l218 to "the example configuration"